src/components/groupList.js

```diff
@@ -70,6 +70,7 @@ class MiniMediaPlayerGroupList extends LitElement {
       <mmp-group-item
         @change=${this.handleGroupChange}
         .item=${item}
+        .hass=${this.hass}
```

Comment:
Great catch, please also add `hass: {}` in the `MiniMediaPlayerGroupItem` class properties.

```js
  static get properties() {
    return {
      hass: {},
      item: {},
      checked: Boolean,
      disabled: Boolean,
      master: Boolean,
    };
  }
```

Reply:
Done 👍